cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/preview/DefaultPreviewManager.java

```diff
@@ -134,8 +134,7 @@
 
     // TODO make maximum size and expire after write configurable?
     this.appInjectors = CacheBuilder.newBuilder()
-      .maximumSize(10)
-      .expireAfterWrite(15, TimeUnit.MINUTES)
+      .maximumSize(cConf.getInt(Constants.Preview.PREVIEW_CACHE_SIZE, 10))
```

Comment:
Since we currently do not support preview in distributed mode, I think it is better to put it in the cdap-site.xml, which specifies this is the property for sdk. In the future, when we have preview in distributed mode, I think we can move this to cdap-default.xml